testdata/ppc-expected-profiles/profile1.yaml

```diff
@@ -8,8 +8,6 @@ spec:
     reserved: 0,2,40,42
   machineConfigPoolSelector:
     machineconfiguration.openshift.io/role: worker-cnf
-  net:
```

Comment:
@swatisehgal I removed the net field completly the profile1.yaml and since there is no user-level-networking argument in profile1.json it describes the exact scenario you were interested in.